mcux/mcux-sdk/drivers/netc/driver_netc.cmake

```diff
@@ -4,6 +4,7 @@ message("driver_netc component is included.")
 
 target_sources(${MCUX_SDK_PROJECT_NAME} PRIVATE
     ${CMAKE_CURRENT_LIST_DIR}/fsl_netc_endpoint.c
+    ${CMAKE_CURRENT_LIST_DIR}/fsl_netc_timer.c
```

Comment:
has this change been submitted to the SDK?

Need to update mcux/README patchlist with this change so we don't overwrite in a future update if the change isn't in the SDK yet.

Reply:
@dleach02 I have just made the changes merged into mcu sdk.
Thanks.